src/lib/components/Toast.svelte

```diff
@@ -27,7 +27,7 @@
       ["error"]: IconError,
       ["info"]: IconInfo,
       ["custom"]: undefined,
-    }[level]);
+    })[level];
```

Comment:
is this correct? not sure how it relates to the PR

Reply:
It was my lint that changed it.